bpf/Makefile

```diff
@@ -18,8 +18,15 @@ bpf_netdev.o:
 bpf_overlay.o:
 	clang ${CLANG_FLAGS} -c bpf_overlay.c -o $@
 
+LB_OPTIONS = \
+	 \
+	-DLB_L3 \
+	-DLB_L4 \
+	-DLB_L3 -DLB_L4
+
 bpf_lb.o:
-	clang ${CLANG_FLAGS} -c bpf_lb.c -o $@
+	$(foreach OPTS,$(LB_OPTIONS), \
+		clang ${OPTS} ${CLANG_FLAGS} -c bpf_lb.c -o $@;)
```

Comment:
The idea here is to compile with each possible combination of configuration options. I want to replace the dummy header files with this to make sure we can compile each program with each possible combination of options that can be provided.